cbpi4gui/src/components/dashboard/widgets/GrafanaChart.js

```diff
@@ -192,7 +188,7 @@ const GrafanaChart = ({ id }) => {
   return (
     <div className="box">
       {}
-      <iframe key={counter} src={model?.props?.url+"?orgId=1&from="+fromTime+"&to="+toTime+"&panelId="+model?.props?.panelID} width={model?.props?.width} height={model?.props?.height} frameborder="0"></iframe>
+      <iframe key={counter} src={model?.props?.url+"?orgId=1&from="+fromTime+"&to="+toTime+"&panelId="+model?.props?.panelID+"&refresh="+(model.props?.refresh || 10)} width={model?.props?.width} height={model?.props?.height} frameborder="0"></iframe>
```

Comment:
After the refresh-value is a 's' for seconds missing and not all values you type in the settingspage are valid. Only those who are defined in the dashboard-settings under auto-refresh if you choose a setting between the defined, the smallest inteval are used. But the update of the values seems faster so if you push a updated version with the missing s, I will merge the pull request.

Reply:
Ok. PR was update. Please review.